hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -36,7 +35,7 @@
 public class HoodieIndexConfig extends DefaultHoodieConfig {
 
   public static final String INDEX_TYPE_PROP = "hoodie.index.type";
-  public static final String DEFAULT_INDEX_TYPE = HoodieIndex.IndexType.BLOOM.name();
+  public static final String DEFAULT_INDEX_TYPE = AbstractHoodieIndex.IndexType.BLOOM.name();
```

Comment:
we can probably keep IndexType in `HoodieIndex` itself.. as a interface? anyways, I may have some detailed comments like these.. but we can defer them to final review